charts/posit-chronicle/values.yaml

```diff
@@ -78,6 +78,15 @@ pod:
     fsGroup: 1000
     fsGroupChangePolicy: "OnRootMismatch"
 
+resources:
+  requests:
+    memory: "2Gi"
+    cpu: "1000m"
+    ephemeralStorage: "100Mi"
+  limits:
+    memory: "3Gi"
+    cpu: "2000m"
+    ephemeralStorage: "200Mi"
```

Comment:
I imagine other charts are storing this at the root level, but I think it could be nested in `pod` for clarity if we also move the `resources` definition to the pod spec level in the `stateful-set.yaml` as mentioned above.

Reply:
Moved to the `pod` level.